### PR TITLE
[zh] sync admin cluster docs

### PR DESCRIPTION
**Pull request by chenxuc**
related: #29329

sync Chinese files for following:
(10) admin 5
2	2	content/zh/docs/tasks/administer-cluster/access-cluster-services.md
3	0	content/zh/docs/tasks/administer-cluster/certificates.md
2	9	content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md
5	5	content/zh/docs/tasks/administer-cluster/topology-manager.md
7	3	content/zh/docs/tasks/administer-cluster/sysctl-cluster.md
15	13	content/zh/docs/tasks/administer-cluster/securing-a-cluster.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 1]
```diff
@@ -6,20 +6,24 @@ content_type: concept
...
+Kubernetes core components such as the API server, scheduler, and controller-manager run on a control plane node. 
+However, add-ons must run on a regular cluster node.
 Some of these add-ons are critical to a fully functional cluster, such as metrics-server, DNS, and UI.
 A cluster may stop working properly if a critical add-on is evicted (either manually or as a side effect of another operation like upgrade)
 and becomes pending (for example when the cluster is highly utilized and either there are other pending pods that schedule into the space
 vacated by the evicted critical add-on pod or the amount of resources available on the node changed for some other reason).
 -->
-除了在主机上运行的 Kubernetes 核心组件（如 api-server 、scheduler 、controller-manager）之外，还有许多插件，由于各种原因，
-必须在常规集群节点（而不是 Kubernetes 主节点）上运行。
+Kubernetes 核心组件（如 API server 、scheduler 、controller-manager）在控制平面节点上运行。但是插件必须在常规集群节点上运行。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md:16]:
```suggestion
Kubernetes 核心组件（如 API 服务器、调度器、控制器管理器）在控制平面节点上运行。
但是插件必须在常规集群节点上运行。
```

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 2]
```diff
@@ -6,20 +6,24 @@ content_type: concept
...
 其中一些插件对于功能完备的群集至关重要，例如 Heapster、DNS 和 UI。
 如果关键插件被逐出（手动或作为升级等其他操作的副作用）或者变成挂起状态，群集可能会停止正常工作。
 关键插件进入挂起状态的例子有：集群利用率过高；被逐出的关键插件 Pod 释放了空间，但该空间被之前悬决的 Pod 占用；由于其它原因导致节点上可用资源的总量发生变化。
 
-
+<!--
+Note that marking a pod as critical is not meant to prevent evictions entirely; it only prevents the pod from becoming permanently unavailable.
+A static pod marked as critical, can't be evicted. However, a non-static pods marked as critical are always rescheduled.
+-->
+注意，把一个 Pod 标记为关键性并不意味着完全防止该 Pod 被逐出；它只能防止该 Pod 变成永久不可用。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md:25]:
```suggestion
注意，把某个 Pod 标记为关键 Pod 并不意味着完全避免该 Pod 被逐出；它只能防止该 Pod 变成永久不可用。
```

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 3]
```diff
@@ -29,12 +33,9 @@ vacated by the evicted critical add-on pod or the amount of resources available
...
 <!--
-To be considered critical, the pod has to run in the `kube-system` namespace (configurable via flag) and
-* Have the priorityClassName set as "system-cluster-critical" or "system-node-critical", the latter being the highest for entire cluster. Alternatively, you could add an annotation `scheduler.alpha.kubernetes.io/critical-pod` as key and empty string as value to your pod, but this annotation is deprecated as of version 1.13 and will be removed in 1.14.
+To mark a Pod as critical, set priorityClassName for that Pod to `system-cluster-critical` or `system-node-critical`. `system-node-critical` is the highest available priority, even higher than `system-cluster-critical`
 -->
-要将 pod 标记为关键性（critical），pod 必须在 kube-system 命名空间中运行（可通过参数配置）。
-同时，需要将 `priorityClassName` 设置为 `system-cluster-critical` 或 `system-node-critical` ，后者是整个群集的最高级别。
-或者，也可以为 Pod 添加名为 `scheduler.alpha.kubernetes.io/critical-pod`、值为空字符串的注解。
-不过，这一注解从 1.13 版本开始不再推荐使用，并将在 1.14 中删除。
+要将 pod 标记为关键性（critical），设置 Pod 的 priorityClassName 为 `system-cluster-critical` 或者 `system-node-critical`。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md:38]:
```suggestion
要将 Pod 标记为关键性（Critical），设置 Pod 的 priorityClassName 为 `system-cluster-critical` 或者
`system-node-critical`。
```

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 4]
```diff
@@ -132,6 +132,13 @@ To align CPU resources with other requested resources in a Pod Spec, the CPU Man
 参看[控制 CPU 管理策略](/zh/docs/tasks/administer-cluster/cpu-management-policies/).
 {{< /note >}}
 
+<!--
+To align memory (and hugepages) resources with other requested resources in a Pod Spec, the Memory Manager should be enabled and proper Memory Manager policy should be configured on a Node. Examine [Memory Manager](/docs/tasks/administer-cluster/memory-manager/) documentation.
+-->
+{{< note >}}
+为了将 Pod 规范中的内存（和巨页）资源与其他请求资源对齐，内存管理器需要被启用，并且节点上应配置了适当的内存管理器策略。查看 [内存管理器](/zh/docs/tasks/administer-cluster/memory-manager/) 文档。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md:139]:
```suggestion
为了将 Pod 规约中的 memory（和 hugepages）资源与所请求的其他资源对齐，需要启用内存管理器，
并且在节点配置适当的内存管理器策略。
查看[内存管理器](/zh/docs/tasks/administer-cluster/memory-manager/)文档。
```

**Comment by chenxuc** [comment]
@tengqm Thanks for the review. I've updated the doc with your comments.

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 5]
```diff
@@ -132,6 +132,14 @@ To align CPU resources with other requested resources in a Pod Spec, the CPU Man
 参看[控制 CPU 管理策略](/zh/docs/tasks/administer-cluster/cpu-management-policies/).
 {{< /note >}}
 
+<!--
+To align memory (and hugepages) resources with other requested resources in a Pod Spec, the Memory Manager should be enabled and proper Memory Manager policy should be configured on a Node. Examine [Memory Manager](/docs/tasks/administer-cluster/memory-manager/) documentation.
+-->
+{{< note >}}
+为了将 Pod 规约中的 memory（和 hugepages）资源与所请求的其他资源对齐，需要启用内存管理器，
+并且在节点配置适当的内存管理器策略。查看 [内存管理器](/zh/docs/tasks/administer-cluster/memory-manager/) 文档。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md:140]:
```suggestion
并且在节点配置适当的内存管理器策略。查看[内存管理器](/zh/docs/tasks/administer-cluster/memory-manager/)
文档。
```

**Comment by chenxuc** [comment]
@tengqm Thanks again for the review. I've updated the doc with your comments.

**Pull request merged**